index.ts

```diff
@@ -14,20 +14,33 @@ type textNode = {
 
 const checkAltText = lintRule(
   "remark-lint:alt-text",
-  (tree: Node, file: VFile): void => {
+  (tree: Node, file: VFile, options: Config): void => {
+    options = {
+      ...defaultConfig,
+      ...options,
+    };
     const textToNodes: { [alt: string]: textNode[] } = {};
     let imageIsLink = false;
     let hasAltText = false;
 
     const aggregate = (node: textNode, ancestors: textNode[]) => {
       imageIsLink = ancestors.filter((a) => a.type === "link").length > 0;
+
       const { alt } = node;
+
       if (alt) hasAltText = true;
-      if (!alt && !imageIsLink) return;
-      if (!alt && imageIsLink) {
-        file.message(createSuggestion("imageLink"), node);
+
+      if (!alt && !imageIsLink) {
+        const suggestion = altText(undefined, options);
+        if (suggestion) file.message(suggestion, node);
+      }
```

Comment:
Now that we have configuration, I think we can throw messages when the alt text is completely empty.